src/Symfony/Component/Intl/Tests/DateFormatter/Verification/IntlDateFormatterTest.php

```diff
@@ -45,6 +45,8 @@ public function testFormatWithTimezoneFromEnvironmentVariable()
 
     protected function getDateFormatter($locale, $datetype, $timetype, $timezone = null, $calendar = IntlDateFormatter::GREGORIAN, $pattern = null)
     {
+        IntlTestHelper::requireFullIntl($this, '55.1');
```

Comment:
Why is this hard coded to 55.1? Symfony is currently bundled with 57.1, but this change makes that tests can only be run against 55.1.